ow_lander/scripts/all_action_trajectories.py

```diff
@@ -138,38 +138,39 @@ def go_to_XYZ_coordinate(move_arm, cs, goal_pose, x_start, y_start, z_start, app
 
     return plan
 
+
 def go_to_Z_coordinate_dig_circular(move_arm, cs, goal_pose, z_start, approximate=True):
-  """
-  :param approximate: use an approximate solution. default True
-  :type move_arm: class 'moveit_commander.move_group.MoveGroupCommander'
-  :type cs: robot current state
-  :type goal_pose: Pose
-  :type z_start: float
-  :type approximate: bool
-  """
-  
-  move_arm.set_start_state(cs)
-  goal_pose.position.z = z_start
-  
-  goal_pose.orientation.x = goal_pose.orientation.x
-  goal_pose.orientation.y = goal_pose.orientation.y
-  goal_pose.orientation.z = goal_pose.orientation.z
-  goal_pose.orientation.w = goal_pose.orientation.w
-
-  # Ask the planner to generate a plan to the approximate joint values generated
-  # by kinematics builtin IK solver. For more insight on this issue refer to:
-  # https://github.com/nasa/ow_simulator/pull/60
-  if approximate:
-    move_arm.set_joint_value_target(goal_pose, True)
-  else:
-    move_arm.set_pose_target(goal_pose)
+    """
+    :param approximate: use an approximate solution. default True
```

Comment:
The number of described parameters doesn't match the number of the method parameters, also one of them is repeated and out of order.

Reply:
Done!